README.rst

```diff
@@ -10,7 +10,7 @@ creation of service accounts, which you can use to authenticate to Google and ma
 - `Product Documentation`_
 
 .. |ga| image:: https://img.shields.io/badge/support-GA-gold.svg
-   :target: https://github.com/googleapis/google-cloud-python/blob/master/README.rst#general-availability
+   :target: https://github.com/googleapis/google-cloud-python/blob/main/README.rst#general-availability
```

Comment:
I get 404 with the new link

Reply:
Presumably this will resolve itself once that repo is migrated and the  `main` branch is created?

Reply:
It will, but to Tony's point we shouldn't let users see a 404. I'll have it reverted for now.

Reply:
Ah, that's true, I thought the migration will happen simultaneously, but that's somewhat optimistic. :slightly_smiling_face:  Tony is right here.